solution/ui/regulations/eregs-component-lib/src/components/ResourceResults.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+    <div class="resources-results">
+        <slot name="empty-state"></slot>
```

Comment:
Is this file used at all?

Reply:
No, I was doing this one way at first and switched course. Thought i removed it.